tests/checks/performance/performance_report_test.py

```diff
@@ -69,6 +70,26 @@ def test_classification(iris_split_dataset_and_model):
                 metric_col = class_col.loc[class_col['Metric'] == metric]
                 assert_that(metric_col['Value'] , close_to(1, 0.3))
 
+
+def test_classification_binary(iris_dataset_single_class_labeled):
+    # Arrange
+    train, test = train_test_split(iris_dataset_single_class_labeled.data, test_size=0.33, random_state=42)
+    train_ds = iris_dataset_single_class_labeled.copy(train)
```

Comment:
why do you have to copy?

Reply:
It's a way to split to train and test a dataset that arrived un-splitted

Reply:
how is that a split?

Reply:
first I do train_test_split, then I create a dataset from each of the splitted dataframes copying the metadata from the original dataset